packages/rollup-plugin/src/index.ts

```diff
@@ -32,6 +33,7 @@ export interface StylableRollupPluginOptions {
     mode?: 'development' | 'production';
     diagnosticsMode?: DiagnosticsMode;
     resolveNamespace?: typeof resolveNamespaceNode;
+    dts?: boolean;
```

Comment:
We would probably want to give the option to generate `dts` source map like in the CLI.

Reply:
> First of all, thanks for your contribution. This feature sounds nice and is something to consider.
> 
> You can see the implementation we made in the CLI ([here](https://github.com/wix/stylable/blob/baded1f5c7d59d6b6f8669656c72cc65f39ae313/packages/cli/src/build-single-file.ts#L149)). If we want to use the same logic, we need to move this to somewhere that it could be re-used.
> 
> Regardless, we would like to talk with you about your use case and setup.

I 'm sure the CLI performs the same purpose. However, I only want to install the rollup plugin and not the CLI in order to produce DTS.
When I use TypeScript, the DTS comes in handy. I just want to use it right away.